src/rules/font-weight-notation/index.js

```diff
@@ -76,7 +77,7 @@ export default function (expectation, options) {
       const weightValueOffset = decl.value.indexOf(weightValue)
 
       if (expectation === "numeric") {
-        if (!isNumbery(weightValue)) {
+        if (!isNumbery(weightValue) && weightValue !== INHERIT_KEYWORD) {
```

Comment:
The logic to ignore `inherit` can probably be moved up the code and therefore won't need to be repeated. Can it go in the conditional ~line 63, so that `checkWeight()` doesn't even run if the word is `inherit`?